packages/web/components/UserProfile/LogoutButton.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { AiOutlineLogout } from 'react-icons/ai';
+import { useTranslation } from 'react-i18next';
+import { useRouter } from 'next/router';
+import { useAuth } from '../../hooks';
+
+const LogoutButton = ({ cb }) => {
+	const { t } = useTranslation(['profile']);
+	const { logout } = useAuth();
+	const router = useRouter();
+
+	const handleLogout = async () => {
+		await router.push('/');
+		logout();
+		if (cb) cb();
```

Comment:
`cb` já é uma função por `default`, logo não precisa checar se ele existe, mas seria interessante checar se de fato é uma função


```suggestion
		if (typeof cb === 'function' ) cb();
```